tests/test_welopen.py

```diff
@@ -0,0 +1,269 @@
+import datetime
+
+import pandas as pd
+
+WELOPEN_CASES = [
+    (
+        """
+    DATES
```

Comment:
The indent here is not Eclipse-syntax, but I presume it works?

Reply:
Yeah, it works. But there was some variation in the actual indent. All now have 1 space.